easybuild/easyconfigs/s/split-seq/split-seq-20191107-add_pkl_barcodes.patch

```diff
@@ -0,0 +1,17 @@
+--- MANIFEST.in	2019-11-13 15:09:45.174322190 +0100
```

Comment:
please add description and author

Reply:
I made that patch to push it upstream, but in hindsight it is overkill for this easyconfig. That fix can be done with a simple `cp` command in `postinstallcmds`. So I updated the easyconfig to not need any patch files.